docs/tutorials/vagrant-installos.rst

```diff
@@ -0,0 +1,148 @@
+Perform an Unattended OS Install
+=================================
+
+Summary
+------------
+
+In this section, you will leverage RackHD to perform an unattended OS install onto a bare metal server.
```

Comment:
It seems you are still working with vnode, not bare metal sever (as stated here) for bootstrap?